### DISCUSSION TITLE
Changing yazi colors with pywal

### DISCUSSION THREAD

**Discussion by moilong** [sxyazi/yazi#772] [2. Show and tell]
Using yazi in voidlinux, hyprland and foot terminal. Using a script to start yazi with pywal colors.
`#!/bin/sh
   cat ~/.cache/wal/sequences
   yazi`
![2024-03-04T10:47:10,999415271+05:30](https://github.com/sxyazi/yazi/assets/112737385/8cb6984d-7d51-4690-badf-4f24e5013726)
![2024-03-04T10:47:39,157410206+05:30](https://github.com/sxyazi/yazi/assets/112737385/0872ddd5-d61c-4a0b-898a-7f24242d92ab)

**Comment by ChuufMaster** [comment]
This is life changing I didn't know you could use pywal like this 🫠

**Comment by AnirudhG07** [comment]
This is amazing! Can you share how you did it? I am also interested in using it.

**Reply by ChuufMaster** [reply]
I think he did this by using the simple script he described,
`#!/bin/sh cat ~/.cache/wal/sequences yazi`
or you could do the same from the command line with this command
`cat ~/.cache/wal/sequences && yazi`
I think this works because the sequences file changes the colours of the terminal using the colour escape seqences.

**Comment by ram-xv** [comment]
Amazing! Thanks for sharing.

Btw I like your desktop setup.

**Comment by nojvspine** [comment]
Researching any yazi and wallust usages I found this conversation, and realizing that it's not what i wanted I made wallust templates for flavor.toml and tmtheme.xml

[yazi-wallust.zip](https://github.com/user-attachments/files/20821429/yazi-wallust.zip)

Feel free to use or change them as you would like
